protos/followme/followme.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+package mavsdk.rpc.followme;
+
+option java_package = "io.mavsdk.followme";
+option java_outer_classname = "FollowMeProto";
+
+service FollowMeService {
```

Comment:
Please add documentation to that (see [here](https://github.com/mavlink/MAVSDK-Proto/blob/master/protos/telemetry/telemetry.proto#L8-L12) for an example).